srv/isc-agent/plugins/tcp/http/main.py

```diff
@@ -18,6 +18,8 @@
 }
 logger = logging.getLogger(__name__)
 
+sslContext = ssl.DefaultOpenSSLContextFactory(settings.PRIVATE_KEY, settings.CERT_KEY)
```

Comment:
Python variables should be in snake case:
https://betterprogramming.pub/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841

Reply:
We also need to do a check that if there is no file at the location then we dont allow HTTPS to be used.